### PR TITLE
add linworks favepada json

### PR DESCRIPTION
**Pull request by duilchoi**
add favepada json

## Description

hi.
i added linworks favepada json.
please review.

## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/20794

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/linworks/favepada/favepada.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "name":"Linworks Fave PadA",
+    "vendorId":"0x4C58",
+    "productId":"0x000B",
+    "keycodes": ["qmk_lighting"],
+    "menus": ["qmk_rgb_matrix"],
```
Comment (Xelus22) [inline v3/linworks/favepada/favepada.json:6]: https://github.com/qmk/qmk_firmware/pull/22120

Please use this as a reference. Your current QMK firmware does not have ALL animations enabled to use `qmk_rgb_matrix`

**Review by Xelus22 — CHANGES_REQUESTED** [review]

File: v3/linworks/favepada/favepada.json (hunk 1 above)
Comment (duilchoi) [reply v3/linworks/favepada/favepada.json:6]: I will refer to it in the next update.Thank you.

Comment (Xelus22) [reply v3/linworks/favepada/favepada.json:6]: It must be in QMK master first

Comment (Xelus22) [reply v3/linworks/favepada/favepada.json:6]: Or you can just enable the ones you have. Please refer to keychron keyboards in this repository on how

Comment (duilchoi) [reply v3/linworks/favepada/favepada.json]: json file updated!

**Review by Xelus22 — APPROVED** [review]

**Pull request merged**